apps/range-voting/app/script.js

```diff
@@ -1,24 +1,35 @@
 import Aragon from '@aragon/client'
 import { combineLatest } from './rxjs'
 import voteSettings, { hasLoadedVoteSettings } from './utils/vote-settings'
+import AddressBookJSON from '../../address-book/build/contracts/AddressBook.json'
```

Comment:
We should prefer instead to import from our `shared/json-abis` folder to prevent unexpected errors when trying to build this file before that artifact is built.

Reply:
Besides that little details I tried the changes and look great